power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/block_matrix.hpp

```diff
@@ -62,6 +62,8 @@ class Block : public block_trait<T, sym, is_tensor, n_sub_block>::ArrayType {
             return (*this)(get_asym_row_idx<r>(), get_asym_col_idx<c>());
         }
     }
+
+    void clear() { *this = Block<T, sym, is_tensor, n_sub_block>{}; }
```

Comment:
doesn't this work?

```suggestion
    void clear() { *this = {}; }
```

if not, then this is OK